packages/amplify_datastore/lib/amplify_datastore.dart

```diff
@@ -30,13 +30,20 @@ class AmplifyDataStore extends DataStorePluginInterface {
       : super(token: _token, modelProvider: modelProvider);
 
   static AmplifyDataStore _instance = AmplifyDataStoreMethodChannel();
-  static AmplifyDataStoreEventChannel events = AmplifyDataStoreEventChannel();
+  static DataStoreStreamController streamWrapper = DataStoreStreamController();
+  ModelProviderInterface models;
```

Comment:
Is this used anywhere?

Reply:
Nope, it's an artifact.  Removed it. Good catch.